src/sass/layouts/_footer.sass

```
@@ -0,0 +1,95 @@
+@import "./base/variables"
+@import "./base/mixins"
+
+footer
+    .footer__main
+        background-color: $background-grey
+        padding: 60px 10px 0px
+        .footer__top
+            .footer__title
+                @include title
+                display: inline
+                text-align: left
+                margin-bottom: 30px
+            span.icon
```

Tách nhau giữa mỗi block cho dễ nhìn nhé
```suggestion
    .footer__main
        background-color: $background-grey
        padding: 60px 10px 0px
        
        .footer__top
            .footer__title
                @include title
                display: inline
                text-align: left
                margin-bottom: 30px
                
            span.icon
```